views/projects/show.erb

```diff
@@ -3,43 +3,86 @@
     <a href="/projects/<%= @project.id %>/edit">edit</a>
   </h2>
   <p>
-    Client Id: <%= @project.client_id %> <br>
-    Resources: <%= @project.resources %> <br>
+    <% for @client in @clients %>
+      <% if @client.active %>
+      Client:
+        <a href="/clients/<%= @client.id %>" >
+          <%= @client.name %>
+        </a>
+      <% end %>
+    <% end %>
+
+    <br>
     Description: <%= @project.description %> <br>
-    Active: <%= @project.active %><br>
-  </p>
+    Active State: <%= @project.active ? "Active" : "Inactive" %>
+  <br>
+  <br>
+  <h4>Nodes</h4>
+ 
   <table class="table">
       <thead>
         <tr>
-          <th>Client Id</th>
           <th>Name</th>
-          <th>Email</th>
-          <th>Description</th>
+          <th>Type</th>
+          <th>Cluster</th>
           <th>Active</th>
         </tr>
       </thead>
       <tbody>
-        <% for @client in @clients %>
-          <% if @client.active %>
+      <% if @nodes %>
+        <% for @node in @nodes %>
           <tr>
-            <th scope="row"><%= @client.id %></th>
-          <td>
-            <a href="/clients/<%= @client.id %>" >
-              <%= @client.name %>
-            </a>
-          </td>
-          <td>
-            <%= @client.contact_email %>
-          </td>
-          <td>
-            <%= @client.description %>
-          </td>
-          <td>
-            <%= @client.active %>
-          </td>
-        </tr>
+            <td>
+              <a href="/clients/<%= @node.id %>" >
+                <%= @node.name %>
+              </a>
+            </td>
+            <td>
+              <%= @node.type %>
+            </td>
+            <td>
+              <%= @node.cluster %>
+            </td>
+            <td>
+              <%= @node.active %>
```

Comment:
Change this to the same construct used to display Active or Inactive in the client block above, and change this column header to say "Status"